platform/features/table/test/controllers/TelemetryTableControllerSpec.js

```diff
@@ -436,5 +436,28 @@ define(
                 expect(mockScope.$broadcast).toHaveBeenCalledWith("remove:rows", discardedRows);
             });
 
+            describe('when telemetry is added', function () {
+                var testRows;
+                var expectedRows;
+
+                beforeEach(function () {
+                    testRows = [ { a: 0 }, { a: 1 }, { a: 2 } ];
```

Comment:
these break code style (no spaces before opening/closing brackets in array literal), can you tweak?